apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -63,13 +63,10 @@ loop(Socket, Transport, State) ->
                     InfoBin = dproto_tcp:encode_bucket_info(Info),
                     Transport:send(Socket, InfoBin),
```

Comment:
i't dmake sense, you can exclude the module from the elvis.conf to make it not error on here.